### ISSUE TITLE
Connecting to websocket.

### ISSUE THREAD

**Issue by chrisbartoloburlo** [rajasimon/django-real-time#1]
What is the correct URL to connect to the websocket? 

I tried `ws://127.0.0.1:8000/real-time/` but got a 404 not found, and then I tried `ws://127.0.0.1:8000/admin/real-time/` and got a 302 code indicating that it was moved to: `/admin/login/?next=/admin/real-time/`. The latter works, however, I am not receiving the messages. 

To retrieve messages I am using a simple python script: 
```
def on_message(ws, message):
    print("received message")
    print(message)
...
if __name__ == "__main__":
    websocket.enableTrace(True)
    ws = websocket.WebSocketApp("ws://127.0.0.1:8000/admin/login/?next=/admin/real-time/",
                              on_open=on_open,
                              on_message=on_message,
                              on_error=on_error,
                              on_close=on_close)
```

**Comment by rajasimon** [comment]
Yeah I have not updated the code for a while and lot changed in the Django channels ecosystem. I have updated the code and you can check this commit. Now if you fire the server it just works.  Besides can you tell me your project scope may be I can help you in the right direction. 

https://github.com/rajasimon/django-real-time/commit/eeb8537a11b32637ddeaacc8a7640a8bbb88cfae

**Comment by chrisbartoloburlo** [comment]
I am looking to build a REST API that upon a POST request sends an updated list of elements via a websocket. I am planning to build using Django but am having trouble setting it up.

**Comment by rajasimon** [comment]
Okay I think above commit is enough to setup the initial project. You are working around with normal request and response cycle and the security stands point you need make sure few things. 

Upon arrival create the session in the backend send the id to the client. After that clients will call the websocket and pass the id. Once the id has any updates you need to receive it. 

I may over complicate but don't know the full scope of the project so.

**Comment by chrisbartoloburlo** [comment]
I was attempting to trigger the server to send and update with all elements when the POST request is received. Do you know how to do that?

**Comment by rajasimon** [comment]
Yes you can also internal message to Consumer function. As soon as the POST request hit the view function. And from that consumer function you can send a client message.

```
from channels.layers import get_channel_layer

channel_layer = get_channel_layer()
await channel_layer.send("channel_name", {
    "type": "chat.message",
    "text": "Hello there!",
})
```

**Comment by chrisbartoloburlo** [comment]
Would you mind giving a more detailed example? Should I also register the channel name somewhere?
